src/main/java/duke/Command.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Command {
+    protected static String indent = "          ";
+
+
+    public static Deadline addDeadline(String command) {
```

Comment:
Similar to above, you are not really adding a Deadline here and so the name is not that clear. By reading the name I would think you are adding a Deadline to some Collection.